### DISCUSSION TITLE
Unable to Install to Program Files Folder

### DISCUSSION THREAD

**Discussion by SakiSakiSakiSakiSaki** [dbeaver/dbeaver#19190] [Q&A]
### Description

I can't install DBeaver to my Program Files folder, it only allows me to install it to `C:\Users\USER\AppData\Local\DBeaver` without the following error.

![Screenshot](https://user-images.githubusercontent.com/41088844/221091097-f8cddd33-9107-4847-abb4-81a73a996e3f.png)

### DBeaver Version

Community Edition 22.3.5

### Operating System

Windows 10

### Steps to reproduce

1. Attempt to install.
2. Change Destination Folder to your User/Program FIles/ folder.
3. Click Next.
4. Get error.

**Comment by LonwoLonwo** [comment]
Hello @SakiSakiSakiSakiSaki 
DId you try to launch the file as Administrator?

**Comment by SakiSakiSakiSakiSaki** [comment]
This worked, thank you!!!
